### PR TITLE
added section about OWA

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/OfficeDocs-Support/issues/1604

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: Outlook/Client/installation/how-to-remove-get-and-set-up-outlook-mobile-app-on-my-phone-option.md [hunk 1]
```diff
@@ -16,15 +16,16 @@ appliesto:
 - Outlook 2019
 - Outlook 2016
 - Outlook for Office 365
+- Exchange Online
 search.appverid: MET150
 ---
-# How to remove the Get and set up Outlook Mobile app on my phone option from Outlook
+# How to remove the Get and set up Outlook Mobile app on my phone option from Outlook and Outlook on the Web
```
Comment (JohanFreelancer9) [inline Outlook/Client/installation/how-to-remove-get-and-set-up-outlook-mobile-app-on-my-phone-option.md:22]:
```suggestion
# How to remove the Get and set up Outlook Mobile app on my phone option from Outlook and Outlook on the web
```

File: Outlook/Client/installation/how-to-remove-get-and-set-up-outlook-mobile-app-on-my-phone-option.md [hunk 2]
```diff
@@ -16,15 +16,16 @@ appliesto:
...
 
 _Original KB number:_ &nbsp; 4010175
 
 ## About the Get and set up the Outlook Mobile app on my phone option
 
-Microsoft Outlook 2016, Outlook 2019, and Outlook for Office 365 offer an option to set up the Outlook Mobile app for your phone. The option appears in four places in Outlook:
+Microsoft Outlook 2016, Outlook 2019, Outlook for Office 365 and Outlook on the Web (OWA) offer an option to set up the Outlook Mobile app for your phone. The option appears in four places in Outlook:
```
Comment (JohanFreelancer9) [inline Outlook/Client/installation/how-to-remove-get-and-set-up-outlook-mobile-app-on-my-phone-option.md:28]:
```suggestion
Microsoft Outlook 2016, Outlook 2019, Outlook for Office 365, and Outlook on the web (OWA) offer an option to set up the Outlook Mobile app for your phone. The option appears in four places in Outlook:
```

File: Outlook/Client/installation/how-to-remove-get-and-set-up-outlook-mobile-app-on-my-phone-option.md [hunk 3]
```diff
@@ -84,3 +85,14 @@ You can remove this option in the Local Group Policy Editor by setting **Disable
 3. Right-click **Disable Outlook Mobile Hyperlink**, and select **Edit**.
 4. Select **Enabled**, and then select **OK**.
 5. Exit Local Group Policy Editor.
+
+## How to remove this option in Outlook on the Web
```
Comment (JohanFreelancer9) [inline Outlook/Client/installation/how-to-remove-get-and-set-up-outlook-mobile-app-on-my-phone-option.md:89]:
```suggestion
## How to remove this option in Outlook on the web
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: Outlook/Client/installation/how-to-remove-get-and-set-up-outlook-mobile-app-on-my-phone-option.md [hunk 4]
```diff
@@ -84,3 +85,13 @@ You can remove this option in the Local Group Policy Editor by setting **Disable
 3. Right-click **Disable Outlook Mobile Hyperlink**, and select **Edit**.
 4. Select **Enabled**, and then select **OK**.
 5. Exit Local Group Policy Editor.
+
+## How to remove this option in Outlook on the web
+
+You can remove this option using Exchange Online PowerShell:
+
+1. [Connect to Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell)
```
Comment (JohanFreelancer9) [inline Outlook/Client/installation/how-to-remove-get-and-set-up-outlook-mobile-app-on-my-phone-option.md:93]:
```suggestion
1. [Connect to Exchange Online PowerShell](/powershell/exchange/connect-to-exchange-online-powershell).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @MaratMussabekov, one more, please.

**Pull request merged**